lifecycle/README.rst

```diff
@@ -63,19 +59,19 @@ We split the tasks of the talker node into separate pieces and execute them as f
 #. cleanup: We destroy the publisher and timer
 
 The principle is implemented in this demo as the typical talker/listener demo.
-However, imaging a real scenario with attached hardware which may have a rather long booting phase, i.e. a laser or camera.
-One could image bringing up the device driver in the configuring state, start and stop only the publishing of the device's data and only in the cleanup/shutdown phase actually shutdown the device.
+However, imagine a real scenario with attached hardware which may have a rather long booting phase, i.e. a laser or camera.
```

Comment:
"However" would indicate some kind of contrast, but there isn't one. It's superfluous here and it could just say, "Imagine a real..."

Reply:
I didn't make any changes here, as with the previous cleanup I think this is fairly clear now.